CSS/Components/input.css

```diff
@@ -0,0 +1,43 @@
+body {
+  font-family: sans-serif;
+}
+.input-text {
+  padding: 0.8rem;
+  border: 1px solid;
+  border-radius: 4px;
+}
+
+.input {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+.input-text:hover {
+  border-color: var(--primary-color);
+}
+.input-text:focus {
+  border-color: var(--primary-color);
+}
```

Comment:
These two styles can be merged.

Reply:
Have made the necessary changes can you look into it